README.md

```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
    - A robust foundation model, trained on 117 classes encompassing various organs and tumors.
    - The model's training and validation involved CT images from over 4,000 patients, ensuring its high accuracy and reliability for medical image analysis.
 
-### How to Get Started
+4. Dicon-web Setup and Dataset Configuration:
+   - [Dicom-web Overview](#dicomweb-overview)
+   - [Notebook: Dataset Creation and Experiment Selection](https://github.com/NVIDIA/monai-cloud-api/blob/main/notebooks/Dataset%20Creation%20and%20Experiment%20Selection.ipynb)
+
+5. OHIF Plugin and Overview:
+   - [Getting Started with OHIF Plugin](#getting-started-with-ohif-plugin)
+   - [Setup Customized OHIF ](https://github.com/NVIDIA/monai-cloud-api/tree/main/plugins/ohif)
+   - [OHIF Applications and UI Demo](#basic-ohif-instruction)
+   - [Interactive Annotation](#interactive-annotation)
+   - [Notebook: Annotation and OHIF Overview](https://github.com/NVIDIA/monai-cloud-api/blob/main/notebooks/Annotation%20and%20Continuous%20Learning%20Overview.ipynb)
+
+
+## How to Get Started
 
 To begin using the NVIDIA MONAI Cloud APIs, you can sign up for early access by filling out this survey: [NVIDIA MONAI Cloud API Early Access](https://developer.nvidia.com/nvidia-monai-cloud-api-early-access-program/join). 
+
+## Getting Started with OHIF Plugin
+
+MONAI Service provide ready-to-use playgroundw with API server and applications for users. 
+If you are going to build and custimize OHIF viewer and MONAI Service plugin. Please refer to the [setup page](https://github.com/NVIDIA/monai-cloud-api/tree/main/plugins/ohif)
+
+### Dicomweb Overview
+Dataset storage can be variant, in the open-sourced setup. We recommend Orthanc for hosting dicom format medical images. 
+
+### **Data Sources**
+
+To create dataset, then you ready to create a reference object to that dataset by utilizing DICOMWeb. 
+DICOMWeb is a modern web standard for accessing DICOM data. 
+By connecting NVIDIA MONAI Cloud APIs to a DICOMWeb endpoint, 
+you can seamlessly integrate to your data into many modern viewers.
+
+In MONAI Service playground, it will automatically deploy Orthanc dicom-web server. 
+For customized setup for Orthanc, 
+
+**Steps:**
+
+```bash
+# Install Orthanc and DICOMweb plugin
+sudo apt-get install orthanc orthanc-dicomweb -y
+
+# Install Plastimatch
+sudo apt-get install plastimatch -y
```

Comment:
Can these two be merged into one-line?
```
sudo apt-get install orthanc orthanc-dicomweb plastimatch -y
```

Reply:
As far as I can tell, we don't need plastimatch for orthanc, but for the conversion of nifti to dicom?  We can leave it as a separate code block, but we should give me information on why we need it, give an example of using it to convert, and also specify the installation is optional and only required for the conversion of files.